src/mqt/bench/evaluation/evaluation_visualization.ipynb

```diff
@@ -0,0 +1,300 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2d493d7f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pickle\n",
+    "from pathlib import Path\n",
+    "import matplotlib\n",
+    "\n",
+    "font = {\"size\": 12}\n",
+    "matplotlib.rc(\"font\", **font)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "12d027e8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with Path(\"./evaluation_data.pkl\").open(\"rb\") as f:\n",
+    "    eval_data = pickle.load(f)\n",
+    "eval_data = np.array(eval_data)\n",
+    "\n",
+    "(\n",
+    "    benchmarks,\n",
+    "    num_qubits,\n",
+    "    depth,\n",
+    "    num_all_gates,\n",
+    "    num_nonlocal_gates,\n",
+    "    program_communication,\n",
+    "    critical_depth,\n",
+    "    entanglement_ratio,\n",
+    "    parallelism,\n",
+    "    liveness,\n",
+    ") = [eval_data[:, i] for i in range(len(eval_data[0]))]\n",
```

Comment:
I believe this code could be simplified by introducing the TypedDict as proposed above.
Because `eval_data` then is a simple list of these TypedDicts.

Reply:
Improved that.